src/snowflake/snowpark/_internal/compiler/utils.py

```diff
@@ -323,3 +327,75 @@ def get_snowflake_plan_queries(
 def is_active_transaction(session):
     """Check is the session has an active transaction."""
     return session._run_query("SELECT CURRENT_TRANSACTION()")[0][0] is not None
+
+
+def plot_plan_if_enabled(root: TreeNode, filename: str) -> None:
+    """A helper function to plot the query plan tree using graphviz useful for debugging.
+    It plots the plan if the environment variable ENABLE_SNOWFLAKE_OPTIMIZATION_PLAN_PLOTTING
+    is set to true.
```

Comment:
so i just need to call the following in my python code or notebook to get the plot, 
```
os.environ["ENABLE_SNOWFLAKE_OPTIMIZATION_PLAN_PLOTTING"] = str(True)
```
maybe add this in the comment also

Reply:
there are multiple ways to set this environment.
you can also set it in bash

```
$ export ENABLE_SNOWFLAKE_OPTIMIZATION_PLAN_PLOTTING=true
```

or you can run your script with this variable set

```
$ ENABLE_SNOWFLAKE_OPTIMIZATION_PLAN_PLOTTING=true python your_script.py
```

Reply:
added examples